### PR TITLE
[FD-1618] Add run all seed data command

### PR DESCRIPTION
**Pull request by brendagutman**

### CODE REVIEW

File: src/locutus_util/ontology_api_etl.py [hunk 1]
```diff
@@ -92,25 +94,29 @@ def add_manual_ontologies():
...
-    ]
-
-    # Update the required fields for the filtered data
-    for entry in monarch_ols:
-        entry['api_url'] = MONARCH_API_BASE_URL
-        entry['api_id'] = "monarch"
-        entry['api_name'] = "Monarch API"
-        entry['version'] = ""
-
     return monarch_ols
```
Comment (brendagutman) [inline src/locutus_util/ontology_api_etl.py:120]: I hope you don't mind a tag along. This logic was creating the issue we were seeing with the data so I went ahead and fixed it.

Comment (torstees) [reply src/locutus_util/ontology_api_etl.py:120]: looks good

File: src/locutus_util/dispatcher.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    )
+    parser.add_argument(
+        '-o', '--option', 
+        choices=[UPDATE_ONTOLOGY_API, UPDATE_SEED_DATA],
+        required=True,
+        help=(
+            f"{UPDATE_ONTOLOGY_API}: Update Ontology API data.\n"
+            f"{UPDATE_SEED_DATA}: Update Seed Data."
+        )
+    )
```
Comment (torstees) [inline src/locutus_util/dispatcher.py:26]: Let's work in some reasonable defaults for these (probably most if not all of the pieces other than the project)

File: src/locutus_util/helpers.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    """Delete all documents in a collection(removes the collection), including 
+    their sub-files(subcollections and their documents)."""
+    db = firestore.Client()
+    collection_ref = db.collection(collection_name)
+
+    start_time = time.time()
+    time_limit = 60
+
+    # Loop until no documents are left
+    while True:
```
Comment (torstees) [inline src/locutus_util/helpers.py:47]: You probably should review [this note](https://cloud.google.com/firestore/docs/manage-data/delete-data#collections) in the docs about deleting a collection. There is a risk if there are too many items in it that can be overcome by batching.

**Review by torstees — CHANGES_REQUESTED** [review]
This looks good, but there are a few things that should be updated: Default options and batching those collection deletes (inline comments will be clear...hopefully! :) )

File: src/locutus_util/helpers.py (hunk 3 above)
Comment (brendagutman) [reply src/locutus_util/helpers.py:47]: I quoted this comment in the ticket for the 'reset_database' ticket. One of the next PRs.

File: src/locutus_util/dispatcher.py [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    parser.add_argument(
+        '-a', '--action',
+        choices=[FETCH_AND_UPLOAD, UPLOAD_FROM_CSV, UPDATE_CSV],
+        default=UPDATE_CSV,
+        help=(
+            f"{FETCH_AND_UPLOAD}: Fetch data from APIs and upload to Firestore.\n"
+            f"{UPLOAD_FROM_CSV}: Upload data from existing CSV to Firestore.\n"
+            f"{UPDATE_CSV}: Fetch data from APIs and update the CSV file only."
```
Comment (brendagutman) [inline src/locutus_util/dispatcher.py]: change order most used to least used.
change `update_csv` to `fetch to csv`

File: src/locutus_util/dispatcher.py (hunk 2 above)
Comment (brendagutman) [reply src/locutus_util/dispatcher.py:26]: I don't see either of these being a priority/more likely to be run. Maybe I could set something up that would run both scripts? I was thinking we decided ontology_api was not seed data.

**Review by torstees — APPROVED** [review]
This looks good. I like that you moved those constants into a common file.

**Pull request merged**